<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgManageStyle</class>
 <widget class="QDialog" name="DlgManageStyle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>310</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Sans Serif</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Manage style</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="margin">
    <number>4</number>
   </property>
   <item>
    <widget class="QTableWidget" name="TableDevice">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="dragDropOverwriteMode">
      <bool>false</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollPerItem</enum>
     </property>
     <property name="horizontalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="cornerButtonEnabled">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string notr="true">#</string>
      </property>
      <property name="font">
       <font>
        <family>Sans Serif</family>
        <pointsize>8</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Style name</string>
      </property>
      <property name="font">
       <font>
        <family>Sans Serif</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="textAlignment">
       <set>AlignLeft|AlignVCenter</set>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="DBAddDevice">
       <property name="toolTip">
        <string>Create a new equipment which comes to be added in the database.</string>
       </property>
       <property name="text">
        <string>Rename style</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>img/db_add.png</normaloff>img/db_add.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="DBRemoveBT">
       <property name="toolTip">
        <string>Delete current equipment.</string>
       </property>
       <property name="text">
        <string>Remove style</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>img/db_remove.png</normaloff>img/db_remove.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="DBResetBT">
       <property name="toolTip">
        <string>Cancel the modifications that you made on an equipment defined in the default ffDiaporama database</string>
       </property>
       <property name="text">
        <string>Reset style to default</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>img/db_status.png</normaloff>img/db_status.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="OkBt">
       <property name="toolTip">
        <string>Accept modification and exit dialog box</string>
       </property>
       <property name="text">
        <string>&amp;Ok</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>img/action_ok.png</normaloff>img/action_ok.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="CancelBt">
       <property name="toolTip">
        <string>Cancel modification and exit dialog box</string>
       </property>
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>img/action_cancel.png</normaloff>img/action_cancel.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="HelpBT">
       <property name="toolTip">
        <string>Open help page describing this dialog box</string>
       </property>
       <property name="text">
        <string>&amp;Help</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>img/help.png</normaloff>img/help.png</iconset>
       </property>
       <property name="shortcut">
        <string>F1</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
